containers/bash-scheduled-job/app/server.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+### Create the response FIFO
+rm -f response
+mkfifo response
+
+function handle_scheduled_job() {
+  RESPONSE=$(echo -e 'HTTP/1.1 200 OK\r\n');
+  bash script.sh;
+}
+
+function handle_handshake() {
+  RESPONSE=$(echo -e 'HTTP/1.1 200 OK\r\n');
+}
+
+function handleRequest() {
+
+  ## Read request, parse each line and breaks until empty line
+  while read line; do
+    echo "$line"
+    trline=$(echo "$line" | tr -d '[\r\n]')
+
+    [ -z "$trline" ] && break
+
+    HEADLINE_REGEX='(.*?)\s(.*?)\sHTTP.*?'
```

Comment:
Can you add a comment to explain what this is doing? It's not clear what is meant by "headline" here. It looks like it's extracting the HTTP method.